@lambertjosh lint issues:

```
stable/gitlab-ce/values.yaml
  15:2      error    missing starting space in comment  (comments)
  20:2      error    missing starting space in comment  (comments)
```

--- a/stable/gitlab-ce/Chart.yaml
+++ b/stable/gitlab-ce/Chart.yaml
@@ -1,6 +1,7 @@
 name: gitlab-ce
 deprecated: true
 version: 0.2.2
+appVersion: 9.4.1
 description: GitLab Community Edition
 keywords:
 - git

--- a/stable/gitlab-ce/requirements.lock
+++ b/stable/gitlab-ce/requirements.lock
@@ -5,5 +5,5 @@ dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
   version: 0.8.1
-digest: sha256:0f00474de1e8698b18b267f0d42db543c032b26609d6bab4a06e174232cc6ef6
-generated: 2017-09-03T14:17:06.691463708-04:00
+digest: sha256:94bbb4af5c8b8e6114938d06650c9e82b0a80613e0b60a6cdbb5031fb7771ae2
+generated: 2018-09-08T11:28:35.128492897-04:00

--- a/stable/gitlab-ce/values.yaml
+++ b/stable/gitlab-ce/values.yaml
@@ -12,12 +12,12 @@ image: gitlab/gitlab-ce:9.4.1-ce.0
 ## The URL (with protocol) that your users will use to reach the install.
 ## ref: https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
 ##
-#externalUrl: http://your-domain.com/
+# externalUrl: http://your-domain.com/
 
 ## Change the initial default admin password if set. If not set, you'll be
 ## able to set it when you first visit your install.
 ##
-#gitlabRootPassword: ""
+# gitlabRootPassword: ""
 
 ## For minikube, set this to NodePort, elsewhere use LoadBalancer
 ## ref: http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types

--- a/stable/gitlab-ee/Chart.yaml
+++ b/stable/gitlab-ee/Chart.yaml
@@ -1,6 +1,7 @@
 name: gitlab-ee
 deprecated: true
 version: 0.2.2
+appVersion: 9.4.1
 description: GitLab Enterprise Edition
 keywords:
 - git

--- a/stable/gitlab-ee/requirements.lock
+++ b/stable/gitlab-ee/requirements.lock
@@ -5,5 +5,5 @@ dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
   version: 0.8.1
-digest: sha256:0f00474de1e8698b18b267f0d42db543c032b26609d6bab4a06e174232cc6ef6
-generated: 2017-09-03T14:17:07.733582008-04:00
+digest: sha256:94bbb4af5c8b8e6114938d06650c9e82b0a80613e0b60a6cdbb5031fb7771ae2
+generated: 2018-09-08T11:28:47.951673071-04:00

--- a/stable/gitlab-ee/values.yaml
+++ b/stable/gitlab-ee/values.yaml
@@ -12,12 +12,12 @@ image: gitlab/gitlab-ee:9.4.1-ee.0
 ## The URL (with protocol) that your users will use to reach the install.
 ## ref: https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
 ##
-#externalUrl: http://your-domain.com/
+# externalUrl: http://your-domain.com/
 
 ## Change the initial default admin password if set. If not set, you'll be
 ## able to set it when you first visit your install.
 ##
-#gitlabRootPassword: ""
+# gitlabRootPassword: ""
 
 ## For minikube, set this to NodePort, elsewhere use LoadBalancer
 ## ref: http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types

